docker/README.md

```diff
@@ -32,10 +32,10 @@ Once the build finished, you can run the container locally as:
 
 ```shell
 # Option 1: run in the background
-docker run --detach -p 8771:8771 -p 7687:7687 -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 --name mira mira:latest
+docker run --detach -p 8771:8771 -p 7687:7687 -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 --name mira mira_epi_dkg:latest
```

Comment:
This change doesn't make sense in this context since above we are tagging the image as `mira:lates` so this should be reverted for consistency